recipes/proj/all/patches/use-cmake-targets-7.2.1.patch

```diff
@@ -0,0 +1,25 @@
+--- a/src/lib_proj.cmake
++++ b/src/lib_proj.cmake
+@@ -414,19 +414,16 @@ if(USE_THREAD AND Threads_FOUND AND CMAKE_USE_PTHREADS_INIT)
+   target_link_libraries(${PROJ_CORE_TARGET} ${CMAKE_THREAD_LIBS_INIT})
+ endif()
+ 
+-target_include_directories(${PROJ_CORE_TARGET} PRIVATE ${SQLITE3_INCLUDE_DIR})
```

Comment:
Does linking to the component removes the need to specifying include dirs?

Reply:
Yes a target is an abstraction having properties such as include dir, lib name, definitions etc

Reply:
Great, thanks. Cmake functions have misleading names sometimes.